routes/index.js

```diff
@@ -0,0 +1,141 @@
+const express = require('express')
```

Comment:
Routes currently seems a bit messy - I'd like to discuss if we could split them into routes and controllers for the sake of brevity. Here's an example

in app.js I'd link my routers:
```js
const smthRouter = require('./routes/smth');
app.use('/smth', smthRouter);
```

in `./routes/smth.js` I'd link routes with proper method from controllers
```js
const router = express.Router();
const smthController = require('../controllers/smth');

router.get('/', smthController.listSmth);
router.post('/', smthController.createSmth);

module.exports = router;
```

and in controller (e.g. `./controllers/smth.js`) I'd handle the whole "logic" splitted into smaller functions
```js
const Smth = require('../models/smth');

exports.listSmth = (req, res, next) => {...}
exports.createSmth = (req, res, next) => {...}
```

such convention should help us managing the whole app slightly easier :wink:

Reply:
It was on my todo list, i'll change it in near future ;)